### PR TITLE
SC 2.2.5 Thorough Review

### PR DESCRIPTION
**Pull request by nitedog**
Addressing issues from [thorough review](https://www.w3.org/2002/09/wbs/1/WCAGvideos_batch1thorough/results#xq2):
- Clarified the extent of information requested in scene 2
- Emphasized the degree of discouragement in scene 3
- Tweaked scene 1 and 4 to emphasize the design aspect

### CODE REVIEW

File: content/SC-2.2.5.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ This page provides the script for Success Criterion 2.2.5 Re-authenticating of t
...
-| 4 | 0:24 - 0:33 | Some websites and apps save the information he has entered before logging him out. When he logs back in, the information he previously typed has not been lost. | We see the screen as he logs in again, and the form re-appears with all the data that was previously there. |
+| 1 | 0:00 - 0:07 | Gopal has mild dementia. Some websites and apps require information that is difficult for him to remember right away. | Gopal (older male) is engaged while browsing on a tablet computer. At some point his flow is interrupted as he tries to remember information he needs to enter (we see that next). |
+| 2 | 0:07 - 0:15 | For example, this website asks for invoice and shipping addresses, account and item numbers, and other information. It takes him a long time to complete all these entries. | He puts down the tablet computer, and gets up to look into his notes (eg. post-it, note book, papers, etc.). We see a form on the screen with many entries (home address, shipping address, etc.). Gopal completed several entries, with others still remaininging (eg. account number, item numbers, etc.). |
```
Comment (jade-mc) [inline content/SC-2.2.5.md]: Suggestion: _It takes him a long time to enter all of this information._

File: content/SC-2.2.5.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ This page provides the script for Success Criterion 2.2.5 Re-authenticating of t
...
+| 3 | 0:15 - 0:24 | Sometimes he is automatically logged out when he takes too long looking up the information to enter. This is very discouraging for him, and often makes him want to give up. | Some indication that some time has lapsed while he was looking up the information. Just as he finished collecting his notes and sits back down to continue typing, we see the session expire (he gets logged out automatically). He looks annoyed/frustrated. |
```
Comment (jade-mc) [inline content/SC-2.2.5.md]: 'and often makes him want to give up' is a bit long, suggestion: _This is very discouraging and frustrating for him_.

File: content/SC-2.2.5.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ This page provides the script for Success Criterion 2.2.5 Re-authenticating of t
...
+| 4 | 0:24 - 0:33 | Websites and apps are more accessible for Gopal when they save the information he has entered before logging him out. When he logs back in, the information he previously typed is still available. | We see the screen as he logs in again, and the form re-appears with all the data that was previously there. |
```
Comment (jade-mc) [inline content/SC-2.2.5.md:59]: I like available here instead of lost.

File: content/SC-2.2.5.md (hunk 2 above)
Comment (jade-mc) [reply content/SC-2.2.5.md]: I like Kevin's suggestion about the interruption.

File: content/SC-2.2.5.md (hunk 1 above)
Comment (bakkenb) [reply content/SC-2.2.5.md]: +1 to Jade's suggestion here.

**Pull request merged**